src/MSGEQ7.hpp

```diff
@@ -64,7 +64,7 @@ CMSGEQ7(void)
 // IC setup
 //================================================================================
 
-template <uint8_t smooth, uint8_t resetPin, uint8_t strobePin, uint8_t firstAnalogPin, uint8_t ...analogPins>
+template <uint16_t smooth, uint8_t resetPin, uint8_t strobePin, uint8_t firstAnalogPin, uint8_t ...analogPins>
```

Comment:
shouldnt this be MSGEQ7_data_t instead of uint16_t? for all other functions as well.

Reply:
Yes, it should.